src/WebJobs.Script/Diagnostics/FileTraceWriter.cs

```diff
@@ -207,7 +205,7 @@ internal void SetNewLogFile()
 
             // we include a machine identifier in the log file name to ensure we don't have any
             // log file contention between scaled out instances
-            string filePath = Path.Combine(_logFilePath, string.Format(CultureInfo.InvariantCulture, "{0}-{1}-{2}.log", DateTime.UtcNow.ToString("yyyy-MM-ddTHH-mm-ssK"), _instanceId, _processId));
```

Comment:
I'm assuming you had a chat with @davidebbo and @suwatch about this. The process ID was a late change to the file name modification and was something they had requested (apparently something they do in Kudu)